tests/Aspire.Hosting.Tests/MongoDB/AddMongoDBTests.cs

```diff
@@ -2,10 +2,13 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Net.Sockets;
+
 using Aspire.Hosting.MongoDB;
 using Aspire.Hosting.Tests.Utils;
 using Aspire.Hosting.Utils;
+
 using Microsoft.Extensions.DependencyInjection;
+
```

Comment:
```suggestion
using Aspire.Hosting.MongoDB;
using Aspire.Hosting.Tests.Utils;
using Aspire.Hosting.Utils;
using Microsoft.Extensions.DependencyInjection;
```

These newlines are unnecessary.